WindowsDevicePortalWrapper/WindowsDevicePortalWrapper/CertificateHandling.cs

```diff
@@ -19,125 +19,95 @@ namespace Microsoft.Tools.WindowsDevicePortal
     public partial class DevicePortal
     {
         /// <summary>
+        /// Gets or sets handler for untrusted certificate handling
+        /// </summary>
+        public event UnvalidatedCertEventHandler UnvalidatedCert;
+
+        /// <summary>
         /// Gets the root certificate from the device.
         /// </summary>
         /// <returns>The device certificate.</returns>
-        private async Task<X509Certificate2> GetDeviceCertificate()
+        public async Task<X509Certificate2> GetDeviceCertificate()
```

Comment:
This doesn't match the GetDeviceCertificate for UWP - this one GETs a cert from the device itself, while SetDeviceCertificate and UWP GetDeviceCertificate both refer to the cert provided to the IDevicePortalConnection.  Should this be GetRootCertificate?